jr-objects/src/main/java/com/fasterxml/jackson/jr/ob/impl/ValueWriterLocator.java

```diff
@@ -307,6 +309,16 @@ private int _registerWriter(Class<?> rawType, ValueWriter valueWriter) {
                 }
             }
             int typeId = _findSimpleType(type, true);
+            // Give plugin the opportunity to override standard value writer
```

Comment:
Ah. So this call was missing?

Reply:
Yes, the override was never getting called at all for enums specifically. I suspect it is because Bean properties are treated differently than root-level types.